composer.json

```diff
@@ -30,10 +30,10 @@
     }
   ],
   "require": {
-    "php": ">=7.1",
-    "symfony/framework-bundle": "^4.1.2 || ^5.0",
-    "symfony/console": "^4.1.2 || ^5.0",
-    "symfony/process": "^4.1.2 || ^5.0",
+    "php": ">=8",
+    "symfony/framework-bundle": "^5.3",
+    "symfony/console": "^5.3",
+    "symfony/process": "^5.3",
     "resque/php-resque": "^1.3"
```

Comment:
You need to use version dev-main. If not using php 8.3.3 it will throw this error:
`Deprecated: Creation of dynamic property Resque_Redis::$driver is deprecated in /var/www/social-bg/vendor/resque/php-resque/lib/Resque/Redis.php on line 134
PHP Deprecated:  Creation of dynamic property Resque_Redis::$driver is deprecated in /var/www/social-bg/vendor/resque/php-resque/lib/Resque/Redis.php on line 134`

Reply:
Hi there - thanks for the report - this package is pretty much abandoned by me now as I moved from Resque queues to Symfony Messenger backed by redis direct.